third-party/sqlalchemy-enum-tables/enumtables/alembic_ops.py

```diff
@@ -7,53 +7,64 @@
 @Operations.register_operation("enum_insert")
 class EnumInsertOp(MigrateOperation):
 
-	def __init__(self, tablename, data = []):
+	def __init__(self, schema, tablename, data = []):
+		self.schema = schema
 		self.tablename = tablename
 		self.data = data
 
 	@classmethod
-	def enum_insert(cls, operations, tablename, data = []):
-		op = cls(tablename, data)
+	def enum_insert(cls, operations, tablename, data = [], schema=None):
+		op = cls(schema, tablename, data)
 		return operations.invoke(op)
 
 	def reverse(self):
-		return EnumDeleteOp(self.tablename, self.data)
+		return EnumDeleteOp(self.schema, self.tablename, self.data)
+
+	@property
+	def fully_qualified_tablename(self):
+		return "%s.%s" % (self.schema, self.tablename) if self.schema else self.tablename
+
 
 @Operations.register_operation("enum_delete")
 class EnumDeleteOp(MigrateOperation):
 
-	def __init__(self, tablename, data = []):
+	def __init__(self, schema, tablename, data = []):
+		self.schema = schema
 		self.tablename = tablename
 		self.data = data
 
 	@classmethod
-	def enum_delete(cls, operations, tablename, data = []):
-		op = cls(tablename, data)
+	def enum_delete(cls, operations, tablename, data = [], schema=None):
+		op = cls(schema, tablename, data)
 		return operations.invoke(op)
 
 	def reverse(self):
 		return EnumInsertOp(self.tablename, self.data)
 
+	@property
+	def fully_qualified_tablename(self):
+		return "%s.%s" % (self.schema, self.tablename) if self.schema else self.tablename
+
 @Operations.implementation_for(EnumInsertOp)
 def insert(operations, operation):
 	if not operation.data:
 		return
 	values = ', '.join("('" + v + "')" for v in operation.data)
-	txt = 'INSERT INTO {tn} (item_id) VALUES {vl}'.format(tn = operation.tablename, vl = values)
+	txt = 'INSERT INTO {tn} (item_id) VALUES {vl}'.format(tn = operation.fully_qualified_tablename, vl = values)
```

Comment:
same comment as above about bind parameters with these insert statements

Reply:
fixed, and ready for review.